Lib/BIOSLIB/StdBios.c

```diff
@@ -2452,12 +2452,19 @@ _more_inits02::
    ld    hl, GCSRShadow
    ld    (hl), a
 
-   lcall   _getDoublerSetting
-
-   ld      a, l
+   lcall   _getDoublerSetting   ; get doubler setting value into L (h always 0)
+   ld      a, L
+   or      a                    ; update the Zero flag
+   jr      z, .notDoubled
+
+   ld      a, 0x0C              ; value to set both /OE0 and /OE1 early output
+   ioi ld  (MTCR), a            ; first, update /OE0, /OE1 early output enable
+   ld      (MTCRShadow), a

```

Comment:
This change, backported from [Dynamic C PR 30](https://github.com/digidotcom/DCRabbit_10/pull/30), enables "early /OE" before enabling the clock doubler.

Note that Dynamic C 9 *could* use the existing code from  that sets MTCR and its shadow to 0 if the clock doubler is disabled, since there isn't any code earlier in the BIOS attempting to set it (as the Dynamic C 10 patch does for the RCM4110).